extensions/-happen3-/additionalroots.js

```diff
@@ -0,0 +1,110 @@
+// Name: Additional Roots
+// ID: additionalroots
+// Description: A collection of new additional roots for your enjoyment.
+// By: -happen3-
+// Original: -happen3-
+
+// I wonder if this will ever pass the check?
+
+(function (Scratch) {
+  "use strict";
+
+  function AdditionalRoots(runtime) {
+    this.runtime = runtime;
+  }
+
+  AdditionalRoots.prototype.getInfo = function () {
+    return {
+      id: "additionalroots",
```

Comment:
Typically, the extension ID should contain some way to identify you make it. "happenroots" would be a good example here.

Reply:
I see.